### PR TITLE
WIP: Edits for this tutorial

### PR DESCRIPTION
**Pull request by thejoycekung**
TODO:
- [x] Page 1
- [ ] Page 2
- [ ] Page 3

### CODE REVIEW

File: blog/page1.md [hunk 1]
```diff
@@ -1,54 +1,62 @@
-# Creating a File Sharing x Conversion Web App with Azure Applications + Online Convert API + Repl.it
+# Creating a File Sharing and Conversion Web App with Azure Applications, Online Convert API, and Repl.it
 
-> If you were a student, knew a student, or had a student that participated in this year's AP Collegeboard Exams, you may have either witnessed or felt the stress of submitting hand written work *virtually* and within a small time constraint.
-![stress](https://media.giphy.com/media/3o7TKRwpns23QMNNiE/giphy.gif)
+If you or someone you know participated in this year's AP Collegeboard Exams, you probably recognize the stress of submitting hand written work *virtually* within a small time constraint.
+![alt text here](https://media.giphy.com/media/3o7TKRwpns23QMNNiE/giphy.gif)
```
Comment (thejoycekung) [inline blog/page1.md:4]: All of your images should have alt-text on them, please see: https://support.microsoft.com/en-us/office/everything-you-need-to-know-to-write-effective-alt-text-df98f884-ca3d-456c-807b-1a1fa82f5dc2

File: blog/page1.md [hunk 2]
```diff
@@ -1,54 +1,62 @@
...
+![alt text here](https://user-images.githubusercontent.com/69332964/99161767-75194280-26c3-11eb-8ad1-c19d63d37bbb.png)
+![alt text here](https://user-images.githubusercontent.com/69332964/99161780-8cf0c680-26c3-11eb-9bfc-78dc3262b038.png)
 
-2. You will need to upgrade your storage account because Event Grid Subscriptions will only work with a v2 version. Follow this [tutorial](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-upgrade?tabs=azure-portal).
+2. You will need to upgrade your storage account because Event Grid Subscriptions will only work with a v2 version. Follow this [tutorial](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-upgrade?tabs=azure-portal) to upgrade it.
 
+vv This code snippet has absolutely no context? You need to introduce it somehow.
+It would also be preferable to, if possible, to break your code up into several smaller sections, instead of a huge monolith.
+That way it's easier to follow along as a reader.
+Also, please include large code snippets as Github Gists (that way we can also have line numbers), and readers can easily fork them.
```
Comment (thejoycekung) [inline blog/page1.md:59]: Please take note of this.

File: blog/page1.md [hunk 3]
```diff
@@ -117,24 +125,23 @@ async function uploadBlob(img, username, filetype){
 }
 ```
 
-* Notice that we are able to name the file with the user's username by receiving it from the header. *Scroll down to see how we sent the username in the header*
-* The parse-multipart library is being used here to parse the image from the POST request we will later make with the frontend; refer to the documentation linked above.
-* I also have some if logic to determine the filetype (there are definitely so many more efficient ways... 🤭)
-* Take note of the process.env values being assigned to variables, use this [tutorial](https://docs.microsoft.com/en-us/azure/azure-functions/functions-how-to-use-azure-function-app-settings) to add in your own secret *shhhh* strings from your storage container. 
-    * The storage container **is the one you created when you started your Function App.** Navigate to it add find your secret strings here:
+* Notice that we are able to name the file with the user's username by receiving it from the header. *Scroll down to see how we sent the username in the header* <- **This is not clear at all, you need to reference a line number**
```
Comment (thejoycekung) [inline blog/page1.md:128]: Note this comment.

File: blog/page1.md [hunk 4]
```diff
@@ -117,24 +125,23 @@ async function uploadBlob(img, username, filetype){
...
-    ![image](https://user-images.githubusercontent.com/69332964/99161822-ec4ed680-26c3-11eb-8977-f12beb496c24.png)
-    * Keep these safe, and add use them in the corresponding variables in the code.
-    * *You're going to need the first key and connection string*
-* Notice the `uploadBlob()` function! This is what's uploading the parsed image to the specified "images" blob container.
-    * [YouTube Video to help explain](https://youtu.be/Qt_VXM_fml4) the handy dandy library
-<br />
+    ![alt text here](https://user-images.githubusercontent.com/69332964/99161798-ba3d7480-26c3-11eb-8e55-eac4bd4cb174.png)
+    ![alt text here](https://user-images.githubusercontent.com/69332964/99161822-ec4ed680-26c3-11eb-8977-f12beb496c24.png)
+    * Keep these safe, and use the first key and connection string in the corresponding variables in the code.
+* I can't! Give me a line number! -> Notice the `uploadBlob()` function! This is what uploads the parsed image to the specified "images" blob container.
```
Comment (thejoycekung) [inline blog/page1.md:136]: Needs line number

File: blog/page1.md [hunk 5]
```diff
@@ -117,24 +125,23 @@ async function uploadBlob(img, username, filetype){
...
+    * Here's a [YouTube Video to help explain](https://youtu.be/Qt_VXM_fml4) the handy dandy library
```
Comment (thejoycekung) [inline blog/page1.md:137]: What library?

File: blog/page1.md [hunk 6]
```diff
@@ -117,24 +125,23 @@ async function uploadBlob(img, username, filetype){
...
 
 ### Frontend: The webpage
-Next, I created a static HTML page that will accept the image from the user and send to the Azure Function we just coded with some JS.
+Next, I created a static HTML page that will accept the image from the user and send to the Azure Function we just coded using Javascript.
+
+*Note*: I removed unnecessary sections of my code because I wanted to make the webpage ✨*fancy*✨, but you can see the whole thing [here](https://github.com/emsesc/bunnimage/blob/main/upload.html)
 
-*Note: I removed unnecessary sections of my code because I wanted to make the webpage ✨*fancy*✨, but you can see the whole thing [here](https://github.com/emsesc/bunnimage/blob/main/upload.html)*
+vvv Again, please don't post entire files. As a reader they're an awful barrier for entry. Highlight the necessary parts and move on. For example, including the head section is not really necessary.
```
Comment (thejoycekung) [inline blog/page1.md:144]: Comment ^^

File: blog/page1.md [hunk 7]
```diff
@@ -192,19 +199,17 @@ Next, I created a static HTML page that will accept the image from the user and
...
 </html>
 ```
-**Here's what I have here:**
-* Input box for the username (simple and *insecure* auth system)
+**Above we have:**
+* Input box for the username (simple but *insecure* auth system)
 * Button to submit
 
-Now, you may have noticed that I have `<script src="js/upload.js"></script>`, that is where we're heading next...
+Now, you may have noticed that I have `<script src="js/upload.js"></script>`, that is where we're heading next... <-- Where!! Line numbers or snippets, please.
```
Comment (thejoycekung) [inline blog/page1.md:206]: Comment

File: blog/page1.md [hunk 8]
```diff
@@ -192,19 +199,17 @@ Next, I created a static HTML page that will accept the image from the user and
...
 
-### Frontend: The Javascript
+### Frontend: The Javascript <- Be more specific, what is the JS in this section for?
```
Comment (thejoycekung) [inline blog/page1.md:208]: Comment - the title isn't specific

File: blog/page1.md [hunk 9]
```diff
@@ -192,19 +199,17 @@ Next, I created a static HTML page that will accept the image from the user and
...
 
-Simply put, this block of Javascript updates the preview thumbnail while getting the picture, gets the username, and sends them both over to the function we just coded.
-1. `loadFile()` is called when the file input changes to display the thumbnail
-2. `handle()` is called when the file is submitted to POST the image and username
-    * Image is sent in the body, and username is sent as a header
+This block of Javascript updates the preview thumbnail while getting the picture, gets the username, and sends them both over to the function we just coded.
 
+First, `loadFile()` is called when the file input changes to display the thumbnail.
```
Comment (thejoycekung) [inline blog/page1.md:212]: This code sample was relatively easy to split up, so I did it for you. Use this as an example!

File: blog/page1.md [hunk 10]
```diff
@@ -245,32 +253,31 @@ async function handle(event) {
         console.log("Blob has been stored successfully!")
 }
 ```
-> Be sure to change the function url to the one of your upload image Function!
-![image](https://user-images.githubusercontent.com/69332964/99188529-73369a00-272a-11eb-93df-04fdce5381df.png)
+> Be sure to change the function url (**where**) to the one of your upload image Function!
```
Comment (thejoycekung) [inline blog/page1.md:256]: Needs context on **where** to update the URL

File: blog/page1.md [hunk 11]
```diff
@@ -1,54 +1,62 @@
...
-* Register for an account on [Online Convert](https://www.online-convert.com/register) (of course with the free version), as we will be using this API convert our images
 
 ## Step 1: Upload the image ⬆️
 
 ### Creating a Function App
-We're going to have a lot of triggers in this project, so we must get started by [creating a Function App](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-azure-function)! Create the Function App and proceed to create the first HTTP trigger (*this will upload our image*).
-
-**Before we start coding the trigger, though, we need to install some npm packages/libraries.**
-![image](https://user-images.githubusercontent.com/69332964/99189070-59e31d00-272d-11eb-80a4-17444e5fac65.png)
+We're going to have a lot of triggers in this project, so let's get started by [creating a Function App](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-azure-function)! Follow those steps to create the Function App, and then create the first HTTP trigger (*this will upload our image*).
```
Comment (thejoycekung) [inline blog/page1.md:33]: oh also you should note that people should keep track of what their storage account name, and/or resource group name is

**Review by emsesc — APPROVED** [review]

**Pull request merged**